api/src/main/java/org/openmrs/module/insuranceclaims/activator/concept/AbstractConceptSetup.java

```diff
@@ -0,0 +1,55 @@
+package org.openmrs.module.insuranceclaims.activator.concept;
+
+import org.openmrs.Concept;
+import org.openmrs.ConceptName;
+import org.openmrs.api.context.Context;
+
+import javax.transaction.Transactional;
+import java.util.Locale;
+
+@Transactional
```

Comment:
verify if this works

Reply:
I've checked it with TransactionSynchronizationManager and it works.